src/elements/ngm-cam-configuration.ts

```diff
@@ -23,6 +23,19 @@ import './ngm-cam-coordinates';
 
 type LockType = '' | 'elevation' | 'angle' | 'pitch' | 'move';
 
+function heightToValue(height: number): number {
+  const m = 0.5 / (height < 0 ? 30000 : 300000);
+  return m * height + 0.5;
+}
+
+function valueToHeight(value: number): number {
+  if (value < 0.5) {
+    return (30000 / 0.5) * value - 30000;
+  } else {
+    return (300000 / 0.5) * value - 300000;
+  }
+}
```

Comment:
These formulas look esoteric to me. Could you add a comment about what they do?
Concretly, I suppose height is the cartographic height.
But what is value? And its units?

Reply:
comments added